### PR TITLE
Design and logic homepage

### PR DESCRIPTION
**Pull request by hoanghuee**
## Related Tickets
- [Task #38456](https://edu-redmine.sun-asterisk.vn/issues/38456)

## Evidence (Screenshot or Video)
image
image
image

### CODE REVIEW

**Review by quanglx-noovado — APPROVED** [review]

**Comment by hoanghuee** [comment]
ready

**Comment by hoanghuee** [comment]
ready

**Review by ghost — APPROVED** [review]

**Review by huyenttm-0901 — COMMENTED** [review]
conflict

**Comment by hoanghuee** [comment]
ready

**Review by huyenttm-0901 — COMMENTED** [review]
2 commits

**Comment by hoanghuee** [comment]
ready

File: resources/views/user/home.blade.php [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            </div>
+        </div>
+    </section>
+    <section class="section services-section" id="services">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-6">
+                    <div class="section-title">
+                        <h2>@lang('home.wwd')</h2>
+                        <p>@lang('home.wwd1')</p>
```
Comment (huyenttm-0901) [inline resources/views/user/home.blade.php]: đặt tên kiểu gì vậy em? 1 mk em code project à?

File: resources/views/user/home.blade.php [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+            </div>
+            <div class="row">
+                <div class="col-sm-6 col-lg-4">
+                    <div class="feature-box-1">
+                        <div class="icon">
+                            <i class="fa fa-desktop"></i>
+                        </div>
+                        <div class="feature-content">
+                            <h5>@lang('home.feature1')</h5>
+                            <p>@lang('home.feature1_des')</p>
```
Comment (huyenttm-0901) [inline resources/views/user/home.blade.php]: tương tự

**Comment by hoanghuee** [comment]
ready

**Review by ghost — APPROVED** [review]

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**